lib/hammer_cli_foreman/compute_resource/libvirt.rb

```diff
@@ -7,29 +7,32 @@ def name
 
       def compute_attributes
         [
-          ['cpus',   _('Number of CPUs')],
-          ['memory', _('String, amount of memory, value in bytes')]
+          ['cpus',   _('Number of CPUs'), { bold: true }],
+          ['memory', _('String, amount of memory, value in bytes'), { bold: true }],
+          ['cpu_mode', _('Possible values: %{modes}') % { modes: 'default, host-model, host-passthrough' }]
         ]
       end
 
       def host_attributes
         [
-          ['start',  _('Boolean (expressed as 0 or 1), whether to start the machine or not')]
+          ['start', _('Boolean (expressed as 0 or 1), whether to start the machine or not')]
         ]
       end
 
       def interface_attributes
         [
           ['type',   _('Possible values: %s') % 'bridge, network'],
           ['bridge', _('Name of interface according to type')],
-          ['model',  _('Possible values: %s') % 'virtio, rtl8139, ne2k_pci, pcnet, e1000']
+          ['model',  _('Possible values: %s') % 'virtio, rtl8139, ne2k_pci, pcnet, e1000'],
```

Comment:
In libevirt we have these three options: 
![image](https://user-images.githubusercontent.com/5609929/63028921-5290ec00-beb8-11e9-84ef-dea25cba35ae.png)
i'm not sure about the bridge, do you know why it's not in the UI is well?

Reply:
It is, when you're creating a host via libvirt and go Interfaces -> New/Edit -> Select `Type`.

Also, see https://github.com/theforeman/foreman/blob/develop/app/models/compute_resources/foreman/model/libvirt.rb#L224-L228. I guess the API does support it.